I added a meow sound for cat and some fixed and added cursor for volume

DMZTdhruv opened the pull request:
**Fixed the cursor pointer glitch for button**, earlier it wasn't showing now it does, and I also **added a cursor for volume**, 
and I added one more cute thing, when you hover over the bottom cat it, the **cat will play meow sound**
I used mp3 because they are of less size compared wav, Wav server better for audio but I guess it is just meow so mp3 will be great 


Hi I am sorry there is something wrong with my indentation I keep 2 tab spaces, I tried to change my tab spaces to match yours but still results in same, this is my first time contributing so my apologies, if possible keep a code format rules😄

kennethnym commented:
omg this is such an adorable pr 😭😭😭😭😭😭 
i think u forgot to upload the meow :3 audio though, can u upload it so that i can try it out? also, self closing tag isn't well supported in html5, so if u could revert those that'd be great!

kennethnym commented:
re ur commit about code format rules, i added `biome.json` to specify the code style i adhere to, u can install biome following [the docs here](https://biomejs.dev/).

DMZTdhruv commented:
> re ur commit about code format rules, i added `biome.json` to specify the code style i adhere to, u can install biome following [the docs here](https://biomejs.dev/).

okay okay sorry, wait i will fix that everything >v<

DMZTdhruv commented:
> re ur commit about code format rules, i added `biome.json` to specify the code style i adhere to, u can install biome following [the docs here](https://biomejs.dev/).

hey so I did made the changes and I can see my most recent commit here too but
I am not sure if you can see it or not?
![Screenshot 2024-07-25 103401](https://github.com/user-attachments/assets/c8b8a4e7-294e-40d2-b78c-4ea131488df4)

I did as you mentioned
**Audio File Upload:** I have uploaded the meow-audio file so you can try it out. It should now play correctly when hovering over the cat image.
**Note**: For this I had to remove *mp3 from .gitignore so  make sure to see it, there wasn't a wav audio and for a small meow audio wav size would be too big

**HTML Tag Changes**: I have reverted the self-closing tags to be more compatible with HTML5 standards and for consistency with the rest of the project.

**Code Format:** I noticed you added a biome.json file to specify the code style rules. I have installed Biome and ensured the code followed the specified formatting rules.

kennethnym commented on web/style.css:
```diff
@@ -151,6 +152,10 @@ a {
   display: flex;
 }
 
+.button-container .button {
+  cursor: pointer;
+}
+
```
i prefer the default cursor for the button as it aligns with how the operating system behaves. imo many websites got it wrong (github included) by using pointer cursor for buttons when it should be reserved for links!

DMZTdhruv commented on web/style.css:
```diff
@@ -174,6 +179,7 @@ a {
     appearance: none;
     border: 2px solid var(--text);
     background-color: var(--base);
+    cursor: col-resize;
```
my windows didn't showed so I thought of using that

DMZTdhruv commented on web/style.css:
```diff
@@ -26,6 +26,7 @@
 html,
 body {
   width: 100%;
+  overflow: hidden;
```
yeah

kennethnym reviewed (approved):
LGTM! html formatting is still messed up but i'll fix it.
